tests/test_split_integer.py

```diff
@@ -1,21 +1,52 @@
 from app.split_integer import split_integer
+from random import randint
 
 
 def test_sum_or_the_parts_should_be_equal_to_value():
-    pass
+    # prime, even and odd values
+    tested_values = [13, 14, 15]
+
+    for tested_number in tested_values:
+
+        # testing each possible splitting
+        for pieces in range(1, tested_number + 1):
+
+            splitted_number = split_integer(tested_number, pieces)
+            assert sum(splitted_number) == tested_number
 
 
 def test_should_split_a_number_into_equal_parts_when_value_is_divisible_by_number_of_parts():
```

Comment:
The same for all tests: just write assertions with concrete values that cover the corresponding test case.